frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -168,11 +169,14 @@ const getDeploymentRevision = (obj: K8sResourceKind): number => {
   return revision && parseInt(revision, 10);
 };
 
-const getDeploymentConfigVersion = (obj: K8sResourceKind): number => {
+export const getDeploymentConfigVersion = (obj: K8sResourceKind): number => {
   const version = getAnnotation(obj, DEPLOYMENT_CONFIG_LATEST_VERSION_ANNOTATION);
   return version && parseInt(version, 10);
 };
 
+export const getDeploymentConfigNameForRC = (obj: K8sResourceKind): string =>
+  obj.metadata.ownerReferences.find((ref) => ref.kind === DeploymentConfigModel.kind)?.name;
```

Comment:
I'd use optional chaining here to be safe. `ownerReferences` is not a required field and not all replication controllers are managed by a deployment config.

```suggestion
  obj.metadata.ownerReferences?.find?.((ref) => ref.kind === DeploymentConfigModel.kind)?.name;
```